.circleci/scripts/pantheon/02-init-site-under-test-clone-existing

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Log in via Terminus
+terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
+
+#
+# This script prepares the site-under-test by cloning the database from
+# an existing site.
+#
+# Use EITHER this script OR the re-install-new script; do not run both.
+#
+
+# Create a new multidev site to test on
+terminus -n env:wake "$TERMINUS_SITE.dev"
+terminus -n build:env:create "$TERMINUS_SITE.dev" "$TERMINUS_ENV" --yes --clone-content --notify="$NOTIFY"
```

Comment:
@greg-1-anderson isn't this always creating the multidev from dev/`master` on Pantheon, rather than the code of the branch in the external Git repository?

Reply:
`build:env:create` will first create a new multidev environment from the specified base environment (here, the dev environment), and then will overlay the code found at the cwd in the new multidev.

If a multidev of the requested name already exists, then it will be re-used, and again, the code from the cwd will be pushed to this multidev.

Reply:
> and then will overlay the code found at the cwd in the new multidev.
Ah, I was forgetting about that part.

Reply:
It still seems `--notify="$NOTIFY"` is running on each commit, even after a multidev exists. Do you think it makes sense to modify `build:env:create` so that `--notify="$NOTIFY"` only fires if the multidev doesn't already exist?

Not having multiple notifications is my only motivation for using [this conditional in `example-wordpress-composer`](https://github.com/pantheon-systems/example-wordpress-composer/blob/master/.circleci/deploy-to-pantheon.sh#L34:L40) rather than just `build:env:create`

Reply:
Let's put that logic in the Build Tools plugin itself: https://github.com/pantheon-systems/terminus-build-tools-plugin/issues/105